### PR TITLE
Page three

### PR DESCRIPTION
**Pull request by luceliu**
Tweaked link hovering, checkboxes, and dropdown menu styles to be as close to Figma spec as possible.

### CODE REVIEW

File: components/Signup/PageThree.vue [hunk 1]
```diff
@@ -1,24 +1,99 @@
...
+      </b-field>
+
+      <div class="field-content">
+        <p>
+          🤖 We participate in Major League Hacking (MLH) as a MLH Member Event and are committed to providing a safe and inclusive environment for everyone attending nwHacks as required by the
+          <a
+            href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf"
+          >MLH Code of Conduct</a>.
+        </p>
+        <b-checkbox v-model.trim="v.hacker.isPrivacyPolicyChecked.$model">
```
Comment (ianmah) [inline components/Signup/PageThree.vue]: are these fields required

File: components/Signup/assets/custom.scss [hunk 2]
```diff
@@ -1,23 +1,74 @@
...
+$secondary: #21D5D4;
 @import "~bulma";
 @import "~buefy/src/scss/buefy";
 
 $background-color: #f2f7fe;
 $font-family: "Apercu Pro", sans-serif;
 $font-color: #21258a;
 
+html {
+  background-color: $background-color;
```
Comment (ianmah) [inline components/Signup/assets/custom.scss]: is this needed? We need to make sure we don't interfere with the main nwHacks site and this is a global css file
I thought setting it for `#signup-form ` was enough?

File: components/Signup/PageThree.vue (hunk 1 above)
Comment (luceliu) [reply components/Signup/PageThree.vue]: u right - i think all the checkboxes are actually? illc heck

File: validators/validators.js [hunk 3]
```diff
@@ -80,33 +95,77 @@ export default {
...
+      }
     },
-    longProject: {
-      required
+    isResumeSharingChecked: {
+      required,
+      checked(v) {
+        return v
+      },
+      firstHackathon: {
```
Comment (ianmah) [inline validators/validators.js]: missing a close bracket `}`

File: components/Signup/PageThree.vue (hunk 1 above)
Comment (ianmah) [reply components/Signup/PageThree.vue]: idk if opt-in for data anonymity should be required but..

File: components/Signup/assets/custom.scss (hunk 2 above)
Comment (luceliu) [reply components/Signup/assets/custom.scss]: WOOPS i added this in for mobile but then realized we need the vertical room for the navbar so i'll take this out 🙃

File: components/Signup/assets/custom.scss [hunk 4]
```diff
@@ -45,11 +93,31 @@ $font-color: #21258a;
 
 #signup-form .field-content {
     display: block;
+    margin: 30px 0;
```
Comment (ianmah) [inline components/Signup/assets/custom.scss]: whats this for?

File: components/Signup/PageThree.vue [hunk 5]
```diff
@@ -1,24 +1,105 @@
...
+            placeholder="Please specify"
+          />
+        </div>
+      </b-field>
+
+      <div class="field-content">
+        <p>
+          🤖 We participate in Major League Hacking (MLH) as a MLH Member Event and are committed to providing a safe and inclusive environment for everyone attending nwHacks as required by the
+          <a
+            href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf"
```
Comment (ianmah) [inline components/Signup/PageThree.vue]: can we make these links new tab (with no ref)

File: components/Signup/assets/custom.scss (hunk 4 above)
Comment (luceliu) [reply components/Signup/assets/custom.scss]: image

File: components/Signup/assets/custom.scss [hunk 6]
```diff
@@ -14,10 +14,58 @@ $font-color: #21258a;
...
+#signup-form .checkbox .control-label {
+  cursor: default;
+}
+
+#signup-form .checkbox input[type=checkbox]:checked ~ .control-label {
+  font-weight: bold;
+}
+
+#signup-form .checkbox + .checkbox {
+  margin-left: 0;
```
Comment (ianmah) [inline components/Signup/assets/custom.scss]: idk if its this line but something is making these checkboxes have funny spacing (page 2)
image

File: validators/validators.js (hunk 3 above)
Comment (luceliu) [reply validators/validators.js]: damn son

File: components/Signup/assets/custom.scss (hunk 4 above)
Comment (ianmah) [reply components/Signup/assets/custom.scss]: image
This adds margin on top as well

File: components/Signup/assets/custom.scss (hunk 6 above)
Comment (luceliu) [reply components/Signup/assets/custom.scss]: ooh okay my bad i'll fix it on my page 👍

File: components/Signup/assets/custom.scss (hunk 4 above)
Comment (ianmah) [reply components/Signup/assets/custom.scss]: Adjust ```#signup-form .field {
  margin: 25px 0;
}``` instead

Comment (luceliu) [reply components/Signup/assets/custom.scss]: oop i meant field

File: components/Signup/assets/custom.scss [hunk 7]
```diff
@@ -89,13 +89,11 @@ $slate-grey: #9195A5;
 }
 
 #signup-form .field {
-  margin: 25px 0;
+  margin: 60px 0;
```
Comment (ianmah) [inline components/Signup/assets/custom.scss]: technically this matches the spec but IMO its way too much..

Comment (luceliu) [reply components/Signup/assets/custom.scss]: @jvssicawu lmao

Comment (ianmah) [reply components/Signup/assets/custom.scss]: (This is the spacing between fields in the form)

Comment (ianmah) [reply components/Signup/assets/custom.scss]: image

Comment (ianmah) [reply components/Signup/assets/custom.scss]: This part looks weird
image

**Pull request merged**